Gemfile.lock

```diff
@@ -310,7 +310,7 @@ GEM
       listen (~> 2.7, >= 2.7.3)
     retriable (1.4.1)
     retryable (1.3.5)
-    rmagick (2.13.2)
+    rmagick (2.13.4)
```

Comment:
Beautiful, 2.13.4 works for me.

For my own education, I don't know best practices on this:  Would it be beneficial to do something like
```
rmagick (>= 2.13.4, < 3.0.0)
```
...assuming we trust the rmagick package to use semantic versioning correctly?  Or is it better practice to be very precise about the versions we depend on?